frontend/server/src/Controllers/Problem.php

```diff
@@ -4742,6 +4743,36 @@ public static function getProblemListForSmarty(
         ];
     }
 
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemListCollection, title: string}, entrypoint: string}
+     *
+     * @omegaup-request-param string $problem_level_tag
+     */
+    public static function getProblemCollectionDetailsForSmarty(
+        \OmegaUp\Request $r
+    ): array {
+        return [
+            'smartyProperties' => [
+                'payload' => [
+                    'levelTags' => \OmegaUp\Controllers\Tag::getLevelTags(),
+                    'problemsAmount' => \OmegaUp\DAO\Problems::getAmountOfProblems(
```

Comment:
nit:

```suggestion
                    'problemCount' => \OmegaUp\DAO\Problems::getProblemsPerTagCount(
```

"amount" se usa para cosas [no-contables](https://ieltsliz.com/uncountable-nouns-word-list/) (como harina, arena, agua, etc.), pero "problemas" es contable, y para eso se prefiere usar "count".